tests/utils.go

```diff
@@ -361,7 +361,10 @@ func DeleteDataVolume(kvClient kubecli.KubevirtClient, namespace, name string) e
 
 func DeleteVirtualMachine(client kubecli.KubevirtClient, namespace, name string) error {
 	return wait.PollImmediate(pollInterval, waitTime, func() (bool, error) {
-		err := client.VirtualMachine(namespace).Delete(name, &metav1.DeleteOptions{})
+		propagationForeground := metav1.DeletePropagationForeground
+		err := client.VirtualMachine(namespace).Delete(name, &metav1.DeleteOptions{
+			PropagationPolicy: &propagationForeground,
```

Comment:
Just out of curiosity, why do we want this? Isn't this the default behavior?

Reply:
Good question, I need to find out what is the default behavior

Reply:
I am sure I have seen tests failed during restore, let me rerun test to check it in an experiment

Reply:
when I remove this, one test fails, when trying to restore, and the PVC was still not removed
https://github.com/brybacki/kubevirt-velero-plugin/blob/dfa59f7e5cdde18b8bd419c8557abcee970600f3/tests/resource_filtering_test.go#L661

Reply:
I suppose more tests can fail, if not checking for deletion of all involved resources

Reply:
Thanks for the clarification!